### ISSUE TITLE
Crit aura suppression for equal level mobs?

### ISSUE THREAD

**Issue by NerdEgghead** [magey/tbc-warrior#53]
The druid Discord has encountered some unusual results in a recent target dummy test. The context of the test was determining whether elemental sharpening stones work while shapeshifted in TBC. Extensive testing with more than 16k samples on Gordok spirits in Classic showed that they do _not_ work in form for Classic. But TBC patch notes indicate that this may have been changed, so we wanted to verify.

A level 60 character on beta autoattacked the level 60 target dummy in cat form 7013 times with no stone applied, and 7167 times with the stone applied. In the first case, 282 crits were measured (4.021%), and in the latter case, 397 crits were measured (5.539%). The spell book crit in cat form was 4.17% with no stone, and 6.17% in cat form with the stone applied. We purposefully took off gear to make the spellbook crit very low to reduce the number of samples required for a significant test.

These are the results of statistical tests on this data:
`The probability that the measured no-stone crit rate deviates from spellbook by at least as much as the measured deviation is 53%.

The probability that the measured with-stone crit rate deviates from (spellbook + 2%) by at least as much as the measured deviation is 2.6%.

The probability that that the measured with-stone crit rate deviates from just spellbook value as much as measured is 6.7x10^-9 .`

So the no-stone results are well within reason, and the with-stone crit is different from no-stone crit at extreme confidence, but the the with-stone crit rate also deviates from the expected value by a statistically significant amount.

If there is some amount of crit suppression for an equal level target, it could explain this behavior. I've read the results on crit aura suppression for +3 level mobs in vanilla; is there any reason to suspect an aura suppression for equal level mobs in TBC?

**Comment by Beannaa** [comment]
Raw logs from Frðllexi#8207:
- no stone
[WoWCombatLog-040721_195324.txt](https://github.com/magey/tbc-warrior/files/6300069/WoWCombatLog-040721_195324.txt)
[WoWCombatLog-041121_194344.txt](https://github.com/magey/tbc-warrior/files/6300070/WoWCombatLog-041121_194344.txt)
- with-stone
[WoWCombatLog-040721_122301.txt](https://github.com/magey/tbc-warrior/files/6300071/WoWCombatLog-040721_122301.txt)
[WoWCombatLog-041121_204545.txt](https://github.com/magey/tbc-warrior/files/6300073/WoWCombatLog-041121_204545.txt)

**Comment by CoolAutumnRain** [comment]
I do not believe we have any testing purposefully done vs. same level mobs in regards to crit suppression.
#1 only has data on +3 or more. 

I did go over the 5 hours of logs for Drakefist Hammers from one of the shaman discords, but the CI is unfortunately too large to draw any meaningful conclusions, but if anything it is pointing in the other direction.

A worthwhile investigation might be returning to the old classic data in regards to ESS however. We have nowhere close to the hits that you do - but as I recall we never got to the point where we could guarantee double ESS giving 4% crit, only that 2 stones give more than 2% crit.
<img width="1066" alt="Screenshot_2020-04-18_at_3 18 59_PM" src="https://user-images.githubusercontent.com/70511316/114471748-6a6ec500-9bf1-11eb-83e1-4afb4020724f.png">
